tests/testEMPsfApprox.py

```diff
@@ -32,43 +33,49 @@
 import lsst.afw.table as afwTable
 import lsst.meas.algorithms as measAlg
 import lsst.meas.base as measBase
-import lsst.meas.base.tests
+from lsst.meas.base.tests import AlgorithmTestCase
 
 import lsst.meas.extensions.ngmix.EMPsfApprox
 
 #   Create an array of size x size containing a 2D circular Gaussian of size sigma.  Normalized to 1.0
+
+
 def makeGaussianArray(size, sigma, xc=None, yc=None):
     if xc == None:
         xc = (size-1)/2.0
     if yc == None:
         yc = (size-1)/2.0
-    image = afwImage.ImageD(afwGeom.Box2I(afwGeom.Point2I(0,0), afwGeom.Point2I(size,size)))
-    array = np.ndarray(shape = (size, size), dtype = np.float64)
-    for yi, yv in enumerate(xrange(0, size)):
-        for xi, xv in enumerate(xrange(0, size)):
+    image = afwImage.ImageD(afwGeom.Box2I(afwGeom.Point2I(0, 0), afwGeom.Point2I(size, size)))
+    array = np.ndarray(shape=(size, size), dtype=np.float64)
+    for yi, yv in enumerate(range(0, size)):
+        for xi, xv in enumerate(range(0, size)):
             array[yi, xi] = np.exp(-0.5*((xv - xc)**2 + (yv - yc)**2)/sigma**2)
     array /= array.sum()
     return array
 
 #   Run a measurement task which has previously been initialized on a single source
+
+
 def runMeasure(task, schema, exposure):
     cat = afwTable.SourceCatalog(schema)
     source = cat.addNew()
     dettask = measAlg.SourceDetectionTask()
 
     # Suppress non-essential task output.
-    dettask.log.setThreshold(dettask.log.WARN)
+    dettask.log.setLevel(dettask.log.WARN)
 
     # We are expecting this task to log an error. Suppress it, so that it
     # doesn't appear on the console or in logs, and incorrectly cause the user
     # to assume a failure.
-    task.log.setThreshold(task.log.FATAL)
+    task.log.setLevel(task.log.FATAL)
     footprints = dettask.detectFootprints(exposure, sigma=4.0).positive.getFootprints()
     source.setFootprint(footprints[0])
     task.run(exposure, cat)
     return source
 
 #   make a Gaussian with one or two components.  Always square of dimensions size x size
```

Comment:
comment to docstring (which lets you remove the two blank lines as well -- also previous).